cmd/concourse/worker.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/tedsuo/ifrit/sigmon"
 )
 
+const SweeperMaxInFlight = 5
```

Comment:
Can we make this a flag instead with 5 as the default? It seems like this could vary based on the user's hardware capabilities.

Reply:
We thought about that, but for example, for volume sweeping, we don't know what strain  might be placed on `btrfs` with parallel sweeping. We thought we should try it out on prod first. 

Would users even try different `SweeperMaxInFlight` values to figure out the optimal one for their setup? In our local tests, setting a bigger number didn't always mean better. But a user might be inclined to believe that raising it makes performance increase.

Reply:
Isn't this ultimately a magic number based on the (I'm assuming few) environments we've tested in?

Having a sensible default based on our testing is great, but forbidding users from configuring it so that it better utilizes their hardware feels like it's going too far.

Once Concourse ships we really have no idea what kind of hardware people will run it on. I'd guess in some cases people will be using pretty powerful machines with fast disks (maybe even ramdisk). They might not even be using `btrfs`, and they might be using a newer kernel than what we've tried. Since all this takes is converting a magic number const to a flag with a default, it seems worth doing if it allows power users to fine-tune things like this without waiting for another Concourse release for us to make it a flag.

If a user sees the flag and increases it expecting performance to improve and it *doesn't*, well, at least now they can eliminate that theory and look for bottlenecks in other areas. (There's also a chance someone will want to *decrease* this, no?)

Reply:
@vito Fair enough. Do we want separate container- and volume-sweeper-max-in-flight values too? Perhaps called VolumeParallelSweepingLimit or something?

Reply:
@ddadlani 👍

How about `--container-sweeper-max-in-flight` and `--volume-sweeper-max-in-flight`. I'm not sure if we need separate flags, but I suppose it doesn't hurt. By my own logic I guess it'd better to have them available. Maybe someone would want to increase one but not the other? 🤔

Reply:
@vito Yeah, if someone is going to tweak them they may as well do so separately. Added the changes in.